swatch-metrics/src/main/resources/application.yaml

```diff
@@ -0,0 +1,137 @@
+"%stage":
+  quarkus:
+    otel:
+      sdk:
+        disabled: false
+"%prod":
+  quarkus:
+    otel:
+      sdk:
+        disabled: false
+
+quarkus:
+  http:
+    port: ${SERVER_PORT}
+    test-port: 0
+  log:
+    level: ${LOGGING_LEVEL_ROOT}
+    category:
+      "com.redhat.swatch":
+        level: ${LOGGING_LEVEL_COM_REDHAT_SWATCH}
+    handler:
+      splunk:
+        enabled: ${ENABLE_SPLUNK_HEC:false}
+        url: ${SPLUNK_HEC_URL:https://splunk-hec.redhat.com:8088/}
+        metadata-source: ${SPLUNK_SOURCE:swatch-metrics}
+        max-retries: ${SPLUNK_HEC_RETRY_COUNT:0}
+        metadata-source-type: ${SPLUNK_SOURCE_TYPE:quarkus_service}
+        metadata-host: ${SPLUNKMETA_host:${USER}@${HOSTNAME}}
+        metadata-fields:
+          namespace: ${SPLUNKMETA_namespace:local}
+        token: ${SPLUNK_HEC_TOKEN:replaceme}
+        include-exception: ${SPLUNK_HEC_INCLUDE_EX:false}
+        batch-size-count: ${SPLUNK_HEC_BATCH_SIZE:1000}
+        format: '%d %-5p [%c{3.}] (%t) %s%e%n'
+        batch-interval: ${SPLUNK_HEC_BATCH_INTERVAL:10S}
+  swagger-ui:
+    path: /api/${quarkus.application.name}/internal/swagger-ui
+    always-include: true
+  smallrye-openapi:
+    path: /api/${quarkus.application.name}/internal/openapi
+  otel:
+    exporter:
+      otlp:
+        endpoint: http://splunk-otel-collector:4317
+    sdk:
+      disabled: true
+  kafka:
+    devservices:
+      enabled: false
+
+# Configuration for profile "metrics-rhel"
+"%metrics-rhel":
+  rhsm-subscriptions:
+    metering:
+      prometheus:
+        metric:
+          queryTemplates:
+            default: >-
+              max(#{metric.prometheus.queryParams[metric]}) by (#{metric.prometheus.queryParams[instanceKey]})
+              * on(#{metric.prometheus.queryParams[instanceKey]}) group_right
+              min_over_time(#{metric.prometheus.queryParams[metadataMetric]}{product="#{metric.prometheus.queryParams[product]}", external_organization="#{runtime[orgId]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}[1h])
+            addonSamples: >-
+              max(#{metric.prometheus.queryParams[metric]}) by (#{metric.prometheus.queryParams[instanceKey]})
+              * on(#{metric.prometheus.queryParams[instanceKey]}) group_right
+              min_over_time(#{metric.prometheus.queryParams[metadataMetric]}{resource_type="addon",resource_name="#{metric.prometheus.queryParams[resourceName]}", external_organization="#{runtime[orgId]}", billing_model="marketplace", support=~"Premium|Standard|Self-Support|None"}[1h])
+          maxAttempts: 50
+          backOffMaxInterval: 50000
+          backOffInitialInterval: 1000
+          backOffMultiplier: 1.5
+          eventSource: ${EVENT_SOURCE:rhelemeter}
+          rangeInMinutes: ${METRICS_RHEL_METERING_RANGE:60}
+          jobMaxAttempts: 3
+          jobBackOffMaxInterval: 50000
+          jobBackOffInitialInterval: 1000
+
+# Configuration for profile "openshift-metering-worker"
```

Comment:
This is a good opportunity for us to rename this profile to `metrics-telemeter`.

Reply:
+1 updated.